test/passing/module_attributes.ml

```diff
@@ -14,5 +14,5 @@ include [%ext] [@warning "item"] [@@warning "structure"]
 
 include (val M) [@warning "item"] [@@warning "structure"]
 
-include ( val Aaaaaaaaaaaaaaaa.Bbbbbbbbbbbbbbbb.Cccccccccccccccc.
-              Dddddddddddddddd ) [@warning "item"] [@@warning "structure"]
+include ( val Aaaaaaaaaaaaaaaa.Bbbbbbbbbbbbbbbb.Cccccccccccccccc
+              .Dddddddddddddddd ) [@warning "item"] [@@warning "structure"]
```

Comment:
A separate issue for later, but it would probably be better to break before the attributes unless its a one-liner